SU-SWS/ansible-playbooks
# SITES-614: add optimizations role to playbook and use pipelines in ansible

# READY FOR REVIEW

# Summary

- Set cache and compression
- Add pipelining in ansible re: https://www.jeffgeerling.com/blog/2017/slow-ansible-playbook-check-ansiblecfg
- Swapped out site exists check from API to a drush call that fetches the sites.json file from the server
- Fixes bug on site_id not being set on new site and custom domain

# Needed By (Date)
- End of sprint

# Urgency
- Every day you let go by is another day you won't get back

# Steps to Test

1. Check out this branch
2. Configure your inventory to migrate a site that 
- hasn't been migrated yet 
- has the performance settings off
- has a custom domain name
3. Run the migration
4. Validate the site migrated and performance settings are on

# Affected Projects or Products
- Migration

# Associated Issues and/or People
- SITES-614
- SITES-695

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
